tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+- name: Set login shell for user www-data
+  user: name=www-data shell=/bin/bash
+
+- name: Clone the opencraft repository
+  git: repo=https://github.com/open-craft/opencraft.git dest="{{ opencraft_root_dir }}" update=no
+  become_user: www-data
+
+- name: Install system dependencies
+  command: make install_system_dependencies chdir="{{ opencraft_root_dir }}"
+
+- name: Install virtualenv
+  pip: name=virtualenv executable=pip3
```

Comment:
Ah, I had actually already posted about this, I thought I hadn't - yup, as mentioned on the task before seeing your reply here, a follow-up task sounds good to me. I'll create the ticket.